Articles/Blog/2023-04-developer-surveys.md

```diff
@@ -8,109 +8,126 @@
+I say "expected" because this is a manifestation of a classic dichotomy at a research organization like an academic institution or a national lab.
+Anywhere shared high performance computing environments are used, there is a dual need to be conservative and reliable, and that is challenged by the constant reality and drive for change and new capabilities.
 
-One of the biggest challenges in this process isn't necessarily knowing what should be worked on, but figuring out how to do that.
-As an example, the RADIUSS team is a small group of people, each only 10-50% of time devoted to RADIUSS, and often in the context of a specific project.
-Even when we know there is a huge problem that needs attention, we might not have the people power to do it.
-So we decided that we would spread the overall messages of the survey to the teams or leaders that might be empowered to do something.
-As an example, one strong piece of insight we got (that is public) is that developers really like VSCode.
-It was very easy for me to see this and pair it with [this announcement from GitLab]([a public facing](https://software.llnl.gov/radiuss/2023/02/08/first-developer-survey/)) and approach one of the leads of the team that maintains GitLab and ask if this could be provided.
-It was also easy to see that many folks were asking for a piece of software to be updated, but then to go to the team responsible and find out that they had good reasons to not be able to do it yet.
+One of the biggest challenges in this process isn't necessarily knowing what are the biggest pain points, but figuring out how to address them with the available resources.
+As an example, the RADIUSS team is small with each member only 10-50% time on the project and often only for a specific component of the whole ecosystem.
+Even when we are aware of big challenges needing attention, it may take such a fragmented team a while to find the resources to fix it.
+So we decided that we would evangelize key takeaways of the survey with teams and project leaders to help inform their priorities going forward.
 
-This leads to my greatest worry about the survey - that we might put it out, get great insights, but not be empowered to take action on many of them.
-I'm hopeful that if we identify many problems, if we can minimally target two to three of them a year to work on for that year, we can slowly make progress toward better developer experience and happiness.
+For example, one key takeaway is that developers really like the [VSCode](https://code.visualstudio.com) editor (not to be confused with the Visual Studio IDE).
+It was very easy to correlate this with [this announcement from GitLab]([a public facing](https://software.llnl.gov/radiuss/2023/02/08/first-developer-survey/)) and approach one of the leads of the team that maintains GitLab and ask if this could be provided.
```

Comment:
Fix link...

```suggestion
It was very easy to correlate this with [this announcement from GitLab](https://software.llnl.gov/radiuss/2023/02/08/first-developer-survey/)) and approach one of the leads of the team that maintains GitLab and ask if this could be provided.
```